alerts/etcdNoLeader.md

```diff
@@ -0,0 +1,38 @@
+# etcdNoLeader
+
+## Meaning
+
+This alert is triggered when etcd cluster does not have a leader for more than 1
+minute.
+
+## Impact
+
+When there is no leader, Kubernetes and OpenShift APIs will not be able to work
+as expected and cluster cannot process any writes, and any write requests are
+queued for processing until a new leader is elected. Operations that preserve
+the health of the workloads cannot be performed.
+
+## Diagnosis
+
+Check the logs of etcd containers to see any further information and to verify
+that etcd does not have leader. Logs should contain something like `etcdserver:
+no leader`. etcd containers are running in the `openshift-etcd`
+namespace/project and in the `etcd` container.
```

Comment:
Should we include a CLI command for this instead? Something the users can directly copy paste and it would work for them without trying to understand many words. Else, what about pointing a generic etcd troubleshooting runbook to just gather logs using `oc adm inspect clusteroperator/etcd` ? We might need it across multiple runbooks.

Reply:
Can add the command sounds great, maybe in follow up and add to all the runbooks at the end if that's okay with you? (as I have two more to add so will refactor them at the end)

Reply:
Oh sure 👍🏼 Totally fine. Sorry I read it later on that this will be done in followups.

Reply:
No all great suggestions!